src/services/ksqldb/serverless.yml

```diff
@@ -33,6 +33,7 @@ custom:
       - val
       - production
   vpc: ${ssm:/aws/reference/secretsmanager/${self:custom.project}/${sls:stage}/vpc, ssm:/aws/reference/secretsmanager/${self:custom.project}/default/vpc}
+  connectImage: ${ssm:/aws/reference/secretsmanager/ecr/images/${self:custom.project}/${self:service}, "confluentinc/ksqldb-server:0.26.0"}
```

Comment:
Please change this parameter name from `connectImage` to something else, like `ksqldbImage`.